src/sentry/migrations/0165_drop_file_indices.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 1.11.29 on 2021-02-24 19:54
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = True
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    # You'll also usually want to set this to `False` if you're writing a data
+    # migration, since we don't want the entire migration to run in one long-running
+    # transaction.
+    atomic = True
```

Comment:
Afaik, this is okay because the migration shouldn't be long-running (already applied on our server and small volumes on self-hosted / st?)

Reply:
I'd still prefer not because there's no value in it. Generally I'd always prefer not using atomic unless there's a reason for it. The migration can be done with SQL to be idempotent. Just a DROP INDEX IF NOT EXISTS iirc. 

On that note, this should be done concurrently just for good measure anyways. Again, there's just no reason to risk it. We do treat Single Tenant as production environments, so if we can mitigate, we should.